src/QuickMarcEditor/QuickMarcEditor.test.js

```diff
@@ -320,14 +334,89 @@ describe('Given QuickMarcEditor', () => {
 
     describe('when delete last 1XX field', () => {
       it('should display edit authority record pane title', () => {
-        const { getByRole, getByText } = renderQuickMarcEditor({
+        const { getAllByRole, getByText } = renderQuickMarcEditor({
           marcType: MARC_TYPES.AUTHORITY,
         });
+        const deleteButtons = getAllByRole('button', { name: 'ui-quick-marc.record.deleteField' });
 
-        fireEvent.click(getByRole('button', { name: 'ui-quick-marc.record.deleteField' }));
+        fireEvent.click(deleteButtons[deleteButtons.length - 1]);
 
         expect(getByText('ui-quick-marc.authority-record.edit.title')).toBeDefined();
       });
     });
   });
+
+  describe('should open remove authority linking modal when record with tag 100 is linked', () => {
```

Comment:
this outer `describe` seems to be not needed - there's already `when` statement in inner `describe` and `should` statement inside